specification/cognitiveservices/ContentSafetyInternal/routes.tsp

```diff
@@ -0,0 +1,121 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/http";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace ContentSafetyInternal;
+
+alias ServiceTraits = SupportsRepeatableRequests &
+SupportsConditionalRequests &
+SupportsClientRequestId;
+
+
+interface BlockOps
+  extends Azure.Core.ResourceOperations<NoRepeatableRequests &
+      NoConditionalRequests &
+      NoClientRequestId> {}
+
+
+interface WhitelistFeature {
+  @route("whitelist/features")
+  @summary("List Whitelist Features.")
+  @doc("Get whitelist features.")
+  op listBlocklistTopTerms is RpcOperation<
```

Comment:
The naming of op seems not correct. `listWritelistFeatures`?

Reply:
Thanks for review. This a PR for internal review. I will abondon the PR to reduce your work.